mimic/model/clb_errors.py

```diff
@@ -128,6 +128,52 @@ def loadbalancer_not_found():
     )
 
 
+def invalid_node_ids_error(node_ids):
+    """
+    Return validation error string given when node_ids are not part of CLB
+
+    :param ``list`` node_ids: List of int node_id
```

Comment:
I am pretty sure that putting markup like that into the middle of a `:param:` is a ReST syntax error.  It should just be `:param list node_ids:`.